assets/css/contents/search/search.scss

```
@@ -0,0 +1,118 @@
+// To use for result display commit 
+// .main__search-results-index {
+// 	.search-results-list {
+// 		@include clearulli;
+// 	}
+// }
+
+.search-bar {
+    width: 100%;
+}
+
+.search-panel {
+	@include columns;
+
+	> :nth-child(1) {
+		@include columns_item(1 / 2);
```

```suggestion
		@include columns_item(math.div(1, 2););
```
Avec le include qui va bien en haut :
`@use "sass:math";`